docker/cassandra/install-cassandra

```diff
@@ -39,10 +42,13 @@ mkdir /etc/cassandra/cassandra-env.sh.d
 mkdir /etc/cassandra/jvm.options.d
 mkdir /etc/cassandra/logback.xml.d
 
-# install cassandra-exporter (Prometheus monitoring support)
-(cd "/usr/share/cassandra/agents" &&
-    curl -SLO "https://github.com/instaclustr/cassandra-exporter/releases/download/v0.9.6/cassandra-exporter-agent-0.9.6.jar" &&
-    ln -s cassandra-exporter-agent-0.9.6.jar cassandra-exporter-agent.jar)
+if [ "$INSTALL_CASSANDRA_EXPORTER" == "true" ]
+then
+    # install cassandra-exporter (Prometheus monitoring support)
+    (cd "/usr/share/cassandra/agents" &&
+        curl -SLO "https://github.com/instaclustr/cassandra-exporter/releases/download/v0.9.6/cassandra-exporter-agent-0.9.6.jar" &&
```

Comment:
@mmonemali I would parametrize also this version 0.9.6 if it is possible.

Reply:
0.9.8 is the latest ;-)